app/Resources/translations/default/AdminCatalogHelp.xlf

```diff
@@ -870,13 +870,18 @@ File: admin-dev/themes/default/template/controllers/categories/helpers/form/form
       <trans-unit id="0205bdbccc495138a109bd8710053565">
         <source>Upload a brand logo from your computer.</source>
         <target>Upload a brand logo from your computer.</target>
-        <note>Line: 58</note>
+        <note>Line: 60</note>
       </trans-unit>
       <trans-unit id="8ed6ad2c4486cdf4c66666671a6d4bf5">
         <source>To add "tags", click inside the field, write something, and then press "Enter".</source>
```

Comment:
Same here, shouldn't it be removed since the updated (right) wording is five lines below?

Reply:
It shouldn't because this wording is not new for 1.7.6, it's from back in 1.7.0: https://github.com/PrestaShop/PrestaShop/commit/1e050e58c0534cd1973a2828548e57680a41d5b7#diff-4aa04a262d7076798f0da887857323ebR259

We said we wouldn't remove wordings from new 1.7 catalogues to avoid breaking modules that are still using them.